configure.ac

```diff
@@ -36,6 +36,27 @@ AC_SUBST([AX_MAJOR_VERSION])
 AC_SUBST([AX_MINOR_VERSION])
 AC_SUBST([AX_POINT_VERSION])
 
+AC_MSG_CHECKING([whether version number is sane])
+AS_IF([printf "%d.%d.%d" ${AX_MAJOR_VERSION} ${AX_MINOR_VERSION} ${AX_POINT_VERSION} >/dev/null 2>&1], [
+    AC_MSG_RESULT([yes])
+],[
+    AC_MSG_RESULT([no])
+    version_err_msg="
+    VERSION ${VERSION} is invalid.
+    Try the following to remedy this:
+    
+      1. Set the variable manually, with FLUX_VERSION=<version>
+         in your environment.
```

Comment:
Setting FLUX_VERSION probably ought to be the last option listed since getting it wrong could cause confusion down the line.